src/WAM/BinaryInfo.ts

```diff
@@ -0,0 +1,12 @@
+import { EventInputType } from './constants'
+
+export class BinaryInfo {
```

Comment:
Hmm, not really. Go ahead and search for class definitions in Baileys, we have a lot (like ObjectRepository)!

This makes the most sense as it provides the best intellisense and tsdoc support!

Reply:
Pretty sure it wasn't added by Adi. He told us on the MD migration that he want to make it more functional. About the tsdocs, Im algo pretty sure it can work the same using functions.